common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -153,7 +153,9 @@
   "feature.wynntils.playerGhostTransparency.name": "Transparent Player Ghost Feature",
   "feature.wynntils.playerGhostTransparency.playerGhostTranslucenceLevel.description": "The level of translucence that should be applied to player ghosts",
   "feature.wynntils.playerGhostTransparency.playerGhostTranslucenceLevel.name": "Player Ghost Translucence",
-  "feature.wynntils.questInfoOverlayFeature.name": "Quest Status Overlay Feature",
+  "feature.wynntils.questInfoOverlay.name": "Quest Status Overlay Feature",
+  "feature.wynntils.questInfoOverlay.overlay.questInfo.description": "Display the currently tracked quest's info",
+  "feature.wynntils.questInfoOverlay.overlay.questInfo.name": "Quest Info Tracker Overlay",
```

Comment:
Why did not `overlay.wynntils.questInfo.name` work? I must admit I'm a bit lost here, but I must say that the new name is quite a mouthful. And also, is this an unrelated fix or something that is needed by the overlay manager?

Reply:
I am pretty sure you wanted this i18n key format for the overlay keys. `  "feature.wynntils.questInfoOverlayFeature.name": "Quest Status Overlay Feature",` is incorrect because it contained the "Feature" part of the feature, which should be left out.

Reply:
So I got only myself to blame for this? :-D

This might have gotten worse since QuestInfoOverlayFeature had "Overlay" in it's name. Maybe we can rename it just "QuestInfoFeature" later on.